sass/_cta.scss

```diff
@@ -0,0 +1,13 @@
+.cta__wrapper {
+  @include component-box;
+  position: fixed;
+  top: 50%;
+  transform: translateY(-50%);
+  right: 0;
+  padding: 0;
+  z-index: 9999;
+  width: 3.75rem;
```

Comment:
Ok but then I will have to add `float:right` on the buttons as well since the wrapper will default to 100% width.